### DISCUSSION TITLE
Note Size Option?

### DISCUSSION THREAD

**Discussion by Adzen927** [KadeArchive/Kade-Engine#1046] [Ideas]
Hi, so I am wondering about an idea and it's about Note Size, usually it's hard for me to play because I can't really register in my brain enough if the notes look jumbled up together and would love a Note Size option so I can actually have a chance to click the notes, it will be pretty useful to some people because they can actually read the notes  and have a chance to click em before they reach and also will you ever add a Custom Keybinds option other than DFJK and WASD, would be pretty useful for some people who don't use those like me, I use QWOP, but in the game I just use AS, Up, Right for my keybinds or switch between that and DFJK. But yeah, I would love a Note Size option and see the notes and have a chance to click them based on the length, a bit hard to try and click all of those, plus I thought I was just bad at FNF but I just actually need note size decreased. I know a lot other people that this will benefit for them aswell. Hope you take my idea into thought. Would be really useful!

**Comment by ACardboardBox1** [comment]
ok.
